pip/utils/outdated.py

```diff
@@ -121,18 +123,21 @@ def pip_version_check(session):
 
         # Refresh the version if we need to or just see if we need to warn
         if pypi_version is None:
-            resp = session.get(
-                PyPI.pip_json_url,
```

Comment:
I'm guessing this ``PyPI.pip_json_url`` can *probably* be deleted from where it is defined now. It'd my guess this is the only place we've used it.